### ISSUE TITLE
Wrong accessibility labels on chat screen

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1867]
Some of the accessibility labels on the chat screen are incorrect. 

## Acceptance Criteria
- '+' button should be labelled 'More'
- 'Send a Proof Request' should be labelled 'Send a proof request'
- '⋮' should be labelled 'Details'

**Comment by jeznorth** [comment]
Verified fixed (IOS 18.4.1 / Android 15)
